server/routerlicious/packages/services-telemetry/src/lumberFormatters.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Lumberjack } from "./lumberjack";
+import { ILumberFormatter, LogLevel } from "./resources";
+import { Lumber } from "./lumber";
+
+export class SanitizationLumberFormatter implements ILumberFormatter {
```

Comment:
Name the file SanitizationLumberFormatter.ts to be consistent with a file per class approach ?

Reply:
Done!